### ISSUE TITLE
About gradient of backbone in pytorch code (Partial FC)

### ISSUE THREAD

**Issue by overpresentme** [deepinsight/insightface#1483]
Hi,

I have question about calculating loss and backward process in partial_fc.py

After calculating softmax loss in "grad[index] -= one_hot", 
it is divided by total batch size for averaging.

I understood up to this point.
But I'm not sure why x_grad is multiplied by world_size after back propagation of softmax loss.

When I delete this line ( "x_grad = x_grad * self.world_size" ) , the train loss is increased and the accuracy is degraded.

So I think it's neccessary work but I don't understand why..
Could you explain this process("x_grad = x_grad * self.world_size")?

Thanks.
